cypress/integration/edit-dataset.spec.js

```diff
@@ -202,6 +202,7 @@ describe('Editing an existing dataset', () => {
     cy.get(`#edit-${resourceToBeEdited}`).click();
     cy.contains('Save');
     cy.get('input[name=resource\\.name]').type(`-updated`);
+    cy.get('input[name=resource\\.url]').type(`-updated`);
```

Comment:
Would this still be a valid URL when you append "-updated"?

Reply:
Yea, I don't see any problem there, the validation also passes